modulefiles/GDAS/orion.lua

```diff
@@ -53,6 +53,9 @@ local mpinproc = '-n'
 setenv('MPIEXEC_EXEC', mpiexec)
 setenv('MPIEXEC_NPROC', mpinproc)
 
+setenv('R2D2_CONFIG', '/work2/noaa/da/cmartin/GDASApp/R2D2_SHARED/config_orion.yaml')
```

Comment:
Not a request for a change, but echoing previous comments expressing need for a central location (e.g., role account) in which to store shared files.

Reply:
Yes, tagging @dtkleist for awareness of a use case